app/models/vm_or_template/right_sizing.rb

```diff
@@ -31,8 +31,57 @@ module VmOrTemplate::RightSizing
 
     virtual_column :max_cpu_usage_rate_average_max_over_time_period,     :type => :float
     virtual_column :max_mem_usage_absolute_average_max_over_time_period, :type => :float
-    virtual_column :cpu_usagemhz_rate_average_max_over_time_period,      :type => :float
-    virtual_column :derived_memory_used_max_over_time_period,            :type => :float
+
+    virtual_attribute :cpu_usagemhz_rate_average_max_over_time_period,
+                      :float, :arel => (lambda do |t|
+                        col = :cpu_usagemhz_rate_average
+                        metric_rollup_table = MetricRollup.arel_table
+                        select_clause = metric_rollup_table[col].maximum
```

Comment:
verify: the only difference between these 2 arel snippets is the column name. (and for others, the use of `max`)

this is not a request for change. I'm finding these very straight forward to read.
(but you can hear the cogs moving in my mind to find an extractable concept)